client/performance-profiler/components/tip/style.scss

```diff
@@ -0,0 +1,22 @@
+.performance-profiler-tip {
+	max-width: 460px;
+	background-color: #e7f0fa;
+	padding: 25px;
+	min-width: 300px;
+
+	h4 {
+		font-size: $font-body-small;
+		font-weight: 500;
+		line-height: $font-title-small;
+		margin-bottom: 10px;
+	}
+
+	p {
+		font-size: $font-body-small;
+		margin-bottom: 0;
+	}
+
+	.learn-more-link {
+		margin-top: 20px;
```

Comment:
Nit: We should hardcode the blueberry color instead of the theme highlight color for this link I think

![CleanShot 2024-08-16 at 16 29 56@2x](https://github.com/user-attachments/assets/5b82fdde-33c2-47f9-960c-07a5cb97501d)

Reply:
I've updated all performance profiler links here https://github.com/Automattic/wp-calypso/pull/93634/commits/157e38b99c995baeb0724bb920e27a25e6376827